packages/terra-compact-interactive-list/src/proptypes/columnShape.js

```diff
@@ -5,22 +5,44 @@ const columnShape = PropTypes.shape({
    * Required string representing a unique identifier for the column.
    */
   id: PropTypes.string.isRequired,
+
   /**
-   * String of text to render within the column header cell.
+   * String of text which will be used to provide a column context and to serve as a column header if needed.
    */
   displayName: PropTypes.string,
+
   /**
-   * Number that specifies the minimum column width in pixels.
+   * Width of the column in width units set by widthUnits prop, defaults to px.
+   * If not set, the column will be considered a flex growing column.
+   * If width is set, but flexGrow prop is also set to true, the width will be used as a flex basis of flex growing column.
    */
-  minimumWidth: PropTypes.number,
+  width: PropTypes.number,
+
+  /**
+   * Minimum width of the column in width units set by widthUnits prop, defaults to px.
+   * Will be disregarded if the width is set and flexGrow is omitted or false.
+   * Is reccomended in cases the column is flex growing column.
+   */
+  mimimumWidth: PropTypes.number,
+
   /**
-   * Number that specifies the maximum column width in pixels.
+   * Maximum width of the column in width units set by widthUnits prop, defaults to px.
+   * Will be disregarded if the width is set and flexGrow is omitted or false.
+   * Is reccomended in cases the column is flex growing column.
    */
   maximumWidth: PropTypes.number,
+
   /**
-   * A number (in px) specifying the width of the column. If not provided, the Table's default column width will be used.
+   * Whether the column width can grow (a flex growing column).
+   * Defaults to false if the width is set and true if the width is not set.
    */
-  width: PropTypes.number,
+  flexGrow: PropTypes.bool,
+
+  /**
+   * Whether the column content should be aligned to the center. If not set to true, the content will be aligned to the left.
+   */
+  alignToCenter: PropTypes.bool,
```

Comment:
Maybe we should just allow the properties of the underlying style.  You could left, center, or right align, most likely.  I would expect the properly to be something like "align" and allow me to specify.

Reply:
renamed the prop to `align` with `center`, `left`, `right` options: https://github.com/cerner/terra-framework/pull/1869/commits/ae408c7c30b997ee75cf380978e4fdc763c5a240